ldms/src/sampler/spank/slurm_notifier.c

```diff
@@ -612,7 +682,49 @@ jbuf_t make_exit_data(spank_t sh, const char *event, const char *context)
 	return jb;
 }
 
-jbuf_t make_task_init_data(spank_t sh, const char *event, const char *context)
+jbuf_t make_step_init_data(spank_t sh, const char *event)
```

Comment:
It is not clear why this function should have an "event" parameter when the only place this is called is at "step_init".